packages/vega-parser/src/parsers/guides/axis-domain.js

```diff
@@ -25,18 +27,27 @@ export default function(spec, config, userEncode, dataRef) {
     strokeOpacity:    _('domainOpacity')
   });
 
-  if (orient === Top || orient === Bottom) {
-    u = 'x';
-    v = 'y';
+  if (!isSignal(spec.orient)) {
```

Comment:
(Minor) Instead of using negation, wouldn't it be easier to read if we flip the if/else?

Reply:
👍